### ISSUE TITLE
Accessibility Issue

### ISSUE THREAD

**Issue by edent** [alphagov/tech-and-data-standards#32]
We're aware that the [Challenge Owners' Guide](https://github.com/alphagov/tech-and-data-standards/blob/master/docs/challenge-owner-guide.md) page contains a graphic which doesn't meet our accessibility requirements. 

This will need to be updated before go-live.

**Comment by Lawrence-G** [comment]
now moved being moved to gov.uk - will not be an issue there
